tests/unit/utils/file-checksum-test.js

```diff
@@ -14,6 +14,8 @@ module('Unit | Utility | file-checksum', function() {
     const fileChecksum = FileChecksum.create({ file: file });
     const checksum = await fileChecksum.createMD5();
     const decodedChecksum = atob(checksum);
+    assert.equal(checksum.length, 24);
+    assert.equal(checksum, 'rL0Y20zC+Fzt72VPzMSk2A==');
```

Comment:
@Glarregle Actually, since the output of `fileChecksum.createMD5()` *is expected to be* a base-64 encoded string, I think that assertions on `checksum` should come first. Assertions on `decodedChecksum` are still good, but they're a plus IMO. 
I would avoid the above assertion because those strange characters might cause issues to developers working on different platforms.

Reply:
perfect :+1: